server/src/controllers/getUsers.js

```diff
@@ -0,0 +1,7 @@
+const { getUsers } = require('../database/queries/get_users');
+
+exports.getUsers = (req, res) => {
+  getUsers().then((users) => {
+    res.json(users.rows);
+  });
```

Comment:
This would throw in unhandledPromiseRejection, make a catch for any suspected errors